docs/docsite/rst/dev_guide/developing_collections_shared.rst

```diff
@@ -65,6 +65,23 @@ By wrapping the import statement in a ``try`` block, the payload builder will om
 
 The optional import behavior also applies to module_utils imported from collections.
 
+.. _collection_dependencies:
+
+Listing collection dependencies
+===============================
+
+We recommend that collections work as standalone, independent units, depending only on ansible-core. However, if your collection must depend on features and functionality from another collection, list the other collection or collections under ``dependencies`` in your collection's :file:`galaxy.yml` file. For more information on the :file:`galaxy.yml` file, see :ref:`collections_galaxy_meta`.
+
+You can use git repositories for collection dependencies during local development and testing. For example:
+
+.. code-block:: yaml
+
+   dependencies: {'git@github.com:organization/repo_name.git': 'devel'}
+
+.. warning
```

Comment:
This is a typo and is parsed as a comment block — it needs to have a trailing `::` to be parsed as an RST directive.